tornado-cash-monitor/src/agent.js

```diff
@@ -0,0 +1,174 @@
+const {
+  Finding, FindingSeverity, FindingType, ethers,
+} = require('forta-agent');
+
+// load config file
+const config = require('../agent-config.json');
+
+// load ABI fragment for tornado proxy contracts
+const { abi } = require('../abi/TornadoProxy.json');
+
+const TORNADO_CASH_ADDRESSES = [
+  '0x722122dF12D4e14e13Ac3b6895a86e84145b6967',
```

Comment:
is there a reason this is hard-coded? granted it's unlikely to change, but not impossible

Reply:
This is the sole Tornado Cash Proxy on mainnet and while it is definitely likely to change at some point, there isn't a simple way to programmatically detect if/when a new Proxy is deployed.